src/MDMTransitionContext.h

```diff
@@ -80,3 +80,9 @@ NS_SWIFT_NAME(TransitionContext)
 @property(nonatomic, strong, readonly, nonnull) UIView *containerView;
 
 @end
+
+NS_SWIFT_NAME(InteractiveTransitionContext)
+@protocol MDMInteractiveTransitionContext
```

Comment:
This means the interactive `start` API won't have access to things like the container view, the transition's direction, or the fore/back view controller - is this intentional? I can imagine that someone might want to have access to those properties in order to be able to configure the interactive behavior.